source/utils/environment.js

```diff
@@ -22,3 +24,23 @@ export function getServicesBaseURL() {
 export function getEventLoggerBase() {
     return `${getServicesBaseURL()}event-logger`;
 }
+
+export function isSandbox() {
+    if (!isBrowser() || isProduction()) {
+        return false;
+    }
+    // fepo
+    if (isBrowser() && get(window, 'window.fandom.environment.environment', '').indexOf('sandbox') > -1) {
```

Comment:
You don't need to check `isBrowser` for each of these; the function will have already returned `false` above when checking `!isBrowser()`.

Reply:
Good point. I had a couple of theories in my head accidentally combined the two. Good catch.